src/main/java/org/ow2/proactive/catalog/service/CatalogObjectService.java

```diff
@@ -305,7 +306,7 @@ private GenericInfoBucketData createGenericInfoBucketData(BucketEntity bucket) {
     public ZipArchiveContent getCatalogObjectsAsZipArchive(String bucketName, List<String> catalogObjectsNames) {
         findBucketByNameAndCheck(bucketName);
         List<CatalogObjectRevisionEntity> revisions = catalogObjectsNames.stream()
-                                                                         .map(name -> catalogObjectRevisionRepository.findDefaultCatalogObjectByNameInBucket(bucketName,
+                                                                         .map(name -> catalogObjectRevisionRepository.findDefaultCatalogObjectByNameInBucket(Arrays.asList(bucketName),
```

Comment:
The called method should not do any changes to your structure? So `Collections.singletonList` might be more appropriate here